include/zephyr/bluetooth/gatt.h

```diff
@@ -1700,6 +1695,11 @@ struct bt_gatt_discover_params;
  *
  *  The attribute object as well as its UUID and value objects are temporary and
  *  must be copied to in order to cache its information.
+ *
+ *  @note @ref bt_gatt_attr is given as an argument to bt_gatt_discover(), but
+ *  it's not a proper object of this type. @ref bt_gatt_attr 's field `perm`,
+ *  and methods read() and write() are not available, and it's unsound to
```

Comment:
I see that `read()` and `write()` does not generate any links to any generic read or write functions, but perhaps we should still avoid the `()` to avoid linking to other functions in the future? 

```suggestion
 *  and methods bt_gatt_attr.read() and bt_gatt_attr.write() are not available, and it's unsound to
```

Should work and add links to the right functions